crates/rspack/tests/tree-shaking/tree-shaking-interop/snapshot/new_treeshaking.snap

```diff
@@ -27,7 +27,7 @@ __webpack_require__.el("./a.js@0:15").then(__webpack_require__.bind(__webpack_re
     console.log(mod);
 });
  const a = "a";
-__webpack_exports__.test = 30;
```

Comment:
This is same as webpack because a harmony export module should not have any cjs export dependency